src/Sarif.UnitTests/RandomSarifLogGenerator.cs

```diff
@@ -47,18 +47,21 @@ public static SarifLog GenerateSarifLogWithRuns(Random randomGen, int runCount)
 
         public static Run GenerateRandomRun(Random random, int? resultCount = null)
         {
-            Run run = new Run() { Resources = new Resources() };
             List<string> ruleIds = new List<string>() { "TEST001", "TEST002", "TEST003", "TEST004", "TEST005" };
             List<Uri> filePaths = GenerateFakeFiles(GeneratorBaseUri, random.Next(20) + 1).Select(a => new Uri(a)).ToList();
-
-            run.Tool = new Tool() { Name = "Test", Version = "1.0", };
-            run.Resources.Rules = GenerateRules(ruleIds);
-            run.Files = GenerateFiles(filePaths);
-
             int results = resultCount == null ? random.Next(100) : (int)resultCount;
-            run.Results = GenerateFakeResults(random, ruleIds, filePaths, results);
 
-            return run;
+            return new Run()
```

Comment:
>Run [](start = 23, length = 3)

just cleaned up object creation to use the standard syntax we prefer these days.

Reply:
Lovely!

---
In reply to: [255249109](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255249109) [](ancestors = 255249109)